dataAccessObjects.py

```diff
@@ -56,6 +56,15 @@ def checkBluetoothIdUsed(self,bluetoothId):
             #TODO raise exception
             return None
 
+    def getUserNickname(self,userId):
+        sql = 'select user_nickname from user where user_id = {userId}'.format(userId=userId)
+        ret = self.db.query(sql)
+        if len(ret):
+            return ret[0][0]
```

Comment:
Yes. I have checked the data type of the result and ensured that is str() type, since I had the same uncertainty of the origin code.